### PR TITLE
fix(rocketmq): allow setting multiple addresses in RocketMQ bridge

### PR DESCRIPTION
**Pull request by lafirest**
Fixes [EMQX-9641](https://emqx.atlassian.net/browse/EMQX-9641)

### <samp>🤖 Generated by Copilot at 6e12abf</samp>

Support multiple RocketMQ servers in the connector config. Rename and update the `server` field and function to `servers` in `emqx_ee_connector_rocketmq.erl`.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: lib-ee/emqx_ee_connector/src/emqx_ee_connector_rocketmq.erl [hunk 1]
```diff
@@ -75,7 +75,7 @@ add_default_fn(OrigFn, Default) ->
         (Field) -> OrigFn(Field)
     end.
 
-server() ->
+servers() ->
     Meta = #{desc => ?DESC("server")},
```
Comment (zmstone) [inline lib-ee/emqx_ee_connector/src/emqx_ee_connector_rocketmq.erl:79]:
```suggestion
    Meta = #{desc => ?DESC("server"), aliases => [server]},
```

Comment (lafirest) [reply lib-ee/emqx_ee_connector/src/emqx_ee_connector_rocketmq.erl:79]: This is a new feature that hasn't been released yet.  So I think there should be no need for an alias

Comment (HJianBo) [inline lib-ee/emqx_ee_connector/src/emqx_ee_connector_rocketmq.erl:79]: nit:
```suggestion
    Meta = #{desc => ?DESC("servers")},
```

**Review by HJianBo — APPROVED** [review]

File: lib-ee/emqx_ee_connector/src/emqx_ee_connector_rocketmq.erl (hunk 1 above)
Comment (lafirest) [reply lib-ee/emqx_ee_connector/src/emqx_ee_connector_rocketmq.erl:79]: will fix this the next PR

**Pull request merged**